packages/web-components/src/components/va-accordion/va-accordion.e2e.ts

```diff
@@ -285,4 +285,16 @@ describe('va-accordion', () => {
       },
     });
   });
+
+  it('shows "Collapse all -" if one or more accordion-items is open on load', async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+      <va-accordion section-heading="The Section Heading">
+        <va-accordion-item header="First item" open="true">Some content</va-accordion-item>
+      </va-accordion>`);
+    
+    const button = await page.find('va-accordion >>> button');
+
+    expect(button.innerText).toEqual('collapse-all -');
```

Comment:
Great idea to add a test! Do we have one yet that confirms mixed `open` states? This one is just testing a single open item.

Reply:
@jamigibbs - good call - I added another test to cover mixed states